django_rest_resetpassword/models.py

```diff
@@ -86,16 +78,16 @@ def get_password_reset_token_expiry_time():
     :return: expiry time
     """
     # get token validation time
-    return getattr(settings, 'DJANGO_REST_MULTITOKENAUTH_RESET_TOKEN_EXPIRY_TIME', 24)
+    return getattr(settings, "DJANGO_REST_MULTITOKENAUTH_RESET_TOKEN_EXPIRY_TIME", 24)
 
 
-def get_password_reset_lookup_field():
+def get_password_reset_lookup_fields():
     """
     Returns the password reset lookup field (default: email)
     Set Django SETTINGS.DJANGO_REST_LOOKUP_FIELD to overwrite this time
     :return: lookup field
     """
-    return getattr(settings, 'DJANGO_REST_LOOKUP_FIELD', 'email')
+    return getattr(settings, "DJANGO_REST_LOOKUP_FIELDS", ["email", "username"])
```

Comment:
default should be `[getattr(settings, "DJANGO_REST_LOOKUP_FIELD", "email")]` to maintain backward compatibility